packages/library/README.md

```diff
@@ -6,861 +6,57 @@
 
 [code-style-prettier-image]: https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square
 [code-style-prettier-url]: https://github.com/prettier/prettier
-[npm-downloads-image]: https://img.shields.io/npm/dm/@solana/web3.js/preview.svg?style=flat
-[npm-image]: https://img.shields.io/npm/v/@solana/web3.js/preview.svg?style=flat
-[npm-url]: https://www.npmjs.com/package/@solana/web3.js/v/preview
+[npm-downloads-image]: https://img.shields.io/npm/dm/@solana/web3.js/experimental.svg?style=flat
+[npm-image]: https://img.shields.io/npm/v/@solana/web3.js/experimental.svg?style=flat
+[npm-url]: https://www.npmjs.com/package/@solana/web3.js/v/experimental
 [semantic-release-image]: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
 [semantic-release-url]: https://github.com/semantic-release/semantic-release
 
-# Solana JavaScript SDK 2.0 Release Candidate
+# @solana/web3.js
 
-If you build JavaScript applications on Solana, it’s likely you’ve worked with `@solana/web3.js` or a library powered by it. With 350K+ weekly downloads on npm, it’s the most-used library in the ecosystem for building program clients, web applications, block explorers, and more.
+This is the JavaScript SDK for building Solana apps for Node, web, and React Native.
 
-Here’s an example of a common code snippet from `@solana/web3.js`:
+## Functions
```

Comment:
Wrote new README content for just the functions specific to `@solana/web3.js`.

Reply:
Thank you! 🫶